### PR TITLE
 Try to fix wishes defined in #232

### PR DESCRIPTION
**Pull request by k-c13**
see #232 

Add function to create random id 	
Add id to constructor 	
Add find function 	
Add exception for already existing id 	
StorageServers::add: Check whether id already exists … 	
Update test 	
Create config class which creates random stuff 	
Implement new config class in SyncSettingItem class

### CODE REVIEW

**Comment by thechauffeur** [comment]
Since #232 is closed now what else should this PR do or can this PR be closed? References to any issues would be nice.

File: src/main/java/de/qabel/core/config/SyncSettingItem.java [hunk 1]
```diff
@@ -5,8 +5,8 @@
 /**
  * Class SyncSettingItem: Class to store common setting for synced item
  */
-public class SyncSettingItem {
-    private int id = 0;
+public class SyncSettingItem extends RandomValues {
```
Comment (cburkert) [inline src/main/java/de/qabel/core/config/SyncSettingItem.java]: I don't think inheritance is the right way here. RandomValues is more a utility class.
Semantically, SyncSettingItem is definitely not a subclass of RandomValues.

File: src/main/java/de/qabel/core/config/RandomValues.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		}
+		return result.toString();
+	}
+
+	/**
+	 * This method generates random integer
+	 * 
+	 * @return Random integer
+	 */
+	private int generateInt(int length) {
```
Comment (cburkert) [inline src/main/java/de/qabel/core/config/RandomValues.java]: What's the point of wrapping Random functions? I don't get it.

Comment (jan-schreib) [inline src/main/java/de/qabel/core/config/RandomValues.java]: The point is to build more possible errors. Nothing more, nothing less.

**Comment by thechauffeur** [comment]
Since #232 is closed and I don't see any issue / discussion for the reason of this PR I will close it now. If someone thinks this is wrong: Open an issue, reference this PR and discuss it. We can reopen this PR if needed.

**Pull request closed without merging**